### PR TITLE
feat[WIP]: Footer

### PR DESCRIPTION
**Pull request by saadahmadhere**

### CODE REVIEW

**Review by abhij1607 — COMMENTED** [review]
Very neat & clean semantic code | | All good practices followed

File: src/Components/Footer/Footer.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.footer_social > * {
+  margin-left: var(--size-3);
+}
+
+.footer_social > *:hover {
+  color: #f9a826;
```
Comment (abhij1607) [inline src/Components/Footer/Footer.css:20]: variable could have been used here

**Review by AnuragSahu11 — APPROVED** [review]
It's good that you are creating small PRs, ready to merge

File: src/Components/Footer/Footer.css (hunk 1 above)
Comment (saadahmadhere) [reply src/Components/Footer/Footer.css:20]: yeah, but I prefer not to use it this time.

**Pull request merged**